.circleci/config.yml

```diff
@@ -17,14 +17,24 @@ jobs:
           name: Run npm install
           command: |
             npm install
+      - run:
+          name: verify error conformance
+          command: |
+            curl -sSL https://github.com/googleapis/gapic-config-validator/releases/download/v0.5.0/gapic-config-validator-0.5.0-linux-amd64.tar.gz > config-validator.tar
+            tar xvf config-validator.tar
+            chmod +x gapic-error-conformance
+            chmod +x build/src/cli.js
+            export PATH=$PATH:`pwd`
```

Comment:
This is very wrong. Never add `pwd` in `$PATH`. Just do `./gapic-error-conformance`.

Reply:
done.